docs/source/using/addons.adoc

```diff
@@ -76,6 +78,32 @@ If the metadata field *OpenShift-Version* is not specified in the add-on header,
 *OpenShift-Version* only supports versions in the form of <major>.<minor>.<patch>.
 ====
 
+[[addon-minishift-version-semantics]]
+== {project}-Version Semantics
```

Comment:
Does the name of this field change between upstream and downstream? Using `{project}` means the content here will change depending on the context. If the field name remains the same, we should use the actual field name.

Reply:
@robin-owen no it will be same so you want it to be hard coded?

Reply:
@praveenkumar Correct -- if it remains the same, there's no use for `{project}`. :)